adal/src/main/java/com/microsoft/aad/adal/AuthenticationResult.java

```diff
@@ -302,6 +342,33 @@ public String getIdToken() {
         return mIdToken;
     }
 
+    /**
+     * Gets if the returned token is valid in terms of extended lifetime
+     *
+     * @return True if the returned token is valid in terms of extended lifetime
+     */
+    final public boolean isExtendedLifeTimeToken() {
+        return mIsExtendedLifeTimeToken;
+    }
+
+    /**
+     * Sets the property isExtendedLifeTimeToken to indicate whether the token
+     * being returned is a token only valid in terms of extended lifetime
+     *
+     * @param isExtendedLifeTimeToken
+     */
+    final void setIsExtendedLifeTimeToken(final boolean isExtendedLifeTimeToken) {
+        this.mIsExtendedLifeTimeToken = isExtendedLifeTimeToken;
```

Comment:
isExtendedLifeTimeToken is used to tell developer that the token is returned as a result of outage mode. 
But @heidijinxujia Shouldn't we expose a get method here? Otherwise how developer would know that the stale token is returned for this reason?